### PR TITLE
Adds Short Delay to Late-Join Job Selection

### PR DESCRIPTION
**Pull request by Krausus**
Now that the late-join job list is significantly larger, I'm anticipating that more people will accidentally click where a job is a split second after it pops up. I *could* avoid this by adding a simple confirmation box to picking a job, but where's the fun in that?

This PR adds a simple bit of Javascript that disables all links for the first second after a window pops up, and includes it with the late-join job list. No more "Surprise! You're playing as the Head of Immediate Regret!"... unless you struggle to accurately click on the links anyway.

The script should actually be generic enough that it could be included in *any* window with potentially unsafe links, though this window is probably the worst one for a rogue click to happen in.

:cl:
tweak: The late-join job listing will now ignore (mis)clicks for the first second after popping up.
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Lmfao, the head of immediate regret-- that's bloody hilarious.

Exact same thing happened to me ages ago when I first started playing, and that's the job I ended up as. Fun times.

**Comment by monster860** [comment]
What head are you talking about, HoP?

**Comment by Krausus** [comment]
> What head are you talking about, HoP?

*Any* head that you pick unintentionally. They're not jobs you can quietly abandon by sneaking off to cryo.

File: html/browser/delay_interactivity.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Disables all links for one second after the browser window opens.
+
+(function(){
+  // If there's already an onload, let's not clobber it
+  var oldonload = window.onload
+  window.onload = function(){
+    if(typeof oldonload == 'function')
+      oldonload();
```
Comment (tigercat2000) [inline html/browser/delay_interactivity.js]: missing {}

Comment (Krausus) [inline html/browser/delay_interactivity.js]: ... it's a single-statement if. You don't need a block for it. And we don't have any coding standards for Javascript!

Comment (tigercat2000) [inline html/browser/delay_interactivity.js]: @Krausus if you're going to do single-statement, put it on the right of the if
don't do this horrific tabbing pls ;W;

Comment (Krausus) [inline html/browser/delay_interactivity.js]: Uuugh, if you're gonna cry about it...

**Comment by Fox-McCloud** [comment]
This doesn't seem necessary to me? I mean, yeah, mistakes may be made initially, but once everyone is used to it, then they have to put up with a delay despite knowing where it is and intentionally meaning to click that very spot.

**Comment by Vivalas** [comment]
A small delay to prevent misclicks isn't at all obstructive and I see no fathomable reason as to why this would be denied. There is no harm in adding this.

**Comment by Krausus** [comment]
> mistakes may be made initially

Mistakes that aren't easy to correct. Best case, you go to the HoP and get re-assigned fairly quickly. Of course, some people will just get frustrated and cryo. Worst case, you accidentally click a head position, and need to call for admin intervention.

> once everyone is used to it, then they have to put up with a delay despite knowing where it is and intentionally meaning to click that very spot

The delay is **one second**. If, from the moment the window pops up, you can find the exact job you want (bear in mind, their position will vary based on what jobs are open), move your mouse over that job, and click, within *one second*, and then proceed to get legitimately annoyed that you have to click a second time, you'd probably ragequit and go back to Counter-Strike within seconds of trying to quickly use any of the in-game interfaces anyway.

**Comment by Regen1** [comment]
Holy shit
It's a second to wait, it is in no way obstructive

:+1: for thinking ahed

**Pull request merged**